### PR TITLE
FMOD Integration Assets

### PR DESCRIPTION
**Pull request by tweexmusic**

### CODE REVIEW

File: Packages/manifest.json [hunk 1]
```diff
@@ -2,12 +2,19 @@
   "dependencies": {
     "com.unity.2d.sprite": "1.0.0",
     "com.unity.2d.tilemap": "1.0.0",
+    "com.unity.ads": "3.7.5",
```
Comment (AnonymousRandomPerson) [inline Packages/manifest.json:5]: Revert this. We don't need these packages.

File: Assets/Resources/BillingMode.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"androidStore":"GooglePlay"}
```
Comment (AnonymousRandomPerson) [inline Assets/Resources/BillingMode.json:1]: Remove this and its meta file.

File: Packages/manifest.json (hunk 1 above)
Comment (tweexmusic) [reply Packages/manifest.json:5]: The entire json, or just that line?

File: Assets/Resources/BillingMode.json (hunk 2 above)
Comment (tweexmusic) [reply Assets/Resources/BillingMode.json:1]: Done!

File: Packages/manifest.json (hunk 1 above)
Comment (AnonymousRandomPerson) [reply Packages/manifest.json:5]: Keep the JSON, but revert all the changes made to the file in this PR.

Comment (tweexmusic) [reply Packages/manifest.json:5]: Done!

**Review by AnonymousRandomPerson — APPROVED** [review]

**Pull request merged**